recipes/uvw/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -Bbuild ${CMAKE_ARGS} -D BUILD_TESTING=OFF -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=$PREFIX .
```

Comment:
uvw seems to provide a shared library. Sholudn't we try to build that one as well?

Reply:
Added